### ISSUE TITLE
Feed generators?

### ISSUE THREAD

**Issue by scripting** [scripting/blue.feedland#6]
@johnspurlock -- is this what you're working on?

https://github.com/bluesky-social/feed-generator

**Comment by johnspurlock** [comment]
yes, that's going to be the way for 3p to contribute new "algorithms" for users to mix and match in their bluesky app - in place of the current "what's hot" tab

That repo has sample code, but it's not necessary to use it (I don't) - you can host code anywhere that can respond to three different http requests

**Comment by johnspurlock** [comment]
I have some basic ones for the unfiltered firehose, but I just had an idea this morning about a great one for news.  Basically posts with links to news stories, but only stories the given user has not seen yet!  Bsky sends you the user in question so I can return different results for different users.
